openreview/conference/helpers.py

```diff
@@ -361,6 +361,7 @@ def get_submission_stage(request_forum, venue):
     abstract_due_date_str = ''
     submission_second_due_date = request_forum.content.get('Submission Deadline', '').strip()
     if submission_second_due_date:
+        submission_deadline_str = submission_second_due_date.strftime('%b %d %Y') + ' UTC-0'
```

Comment:
Could we take the opportunity to change the format here and show the time too? the string should be "%b %d %Y %I:%M%p" like the abstract string. You would need to fix the test too.